include/ros2_socketcan/socket_can_receiver.hpp

```diff
@@ -47,7 +48,7 @@ class SOCKETCAN_PUBLIC SocketCanReceiver
   /// \return The CanId for the received can_frame, with length appropriately populated
   /// \throw SocketCanTimeout On timeout
   /// \throw std::runtime_error on other errors
-  CanId receive(
+  std::tuple<CanId, uint64_t> receive(
```

Comment:
Tuples are cool, but they don't have names for the members. I think this is fine, but would have preferred a simple struct.

Reply:
yeah I opened #12 that packs the time stamp as part of the existing `CanId` object.